openedx/adg/lms/urls.py

```diff
@@ -18,6 +18,11 @@
         r'^api/applications/',
         include('openedx.adg.lms.applications.api_urls', namespace='applications_api')
     ),
+    # ADG webinars app
+    url(
+        r'^webinar/',
+        include('openedx.adg.lms.webinars.urls'),
+    ),
```

Comment:
Can we use [path](https://docs.djangoproject.com/en/3.2/ref/urls/) here instead to url as its syntax is simple fulfills our purpose and [url can be deprecated in future releases](https://docs.djangoproject.com/en/3.0/ref/urls/#url)